### PR TITLE
Using bootstrap to design a responsive web clone of the newsweek website 

### PR DESCRIPTION
**Pull request by alvinlouis29**
this project is aimed to use bootstrap in building a responsive webpage clone of the Newsweek website at www.newsweek.com

### CODE REVIEW

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 


Hello, @alvinlouis29! 
Good start of the project 👍 
Nice set up of bootstrap! 👏 

However, before starting a code review is mandatory to set up the linters, you can find the guidelines [here](https://github.com/microverseinc/linters-config/tree/master/html-css)
Below 👇  I point some of the changes you need to do in order to get the next review approved.
Good luck with the changes 💪 

## Changes
- The title and text on your pull request must be meaningful, please modify it
- Your readme file is good but needs to be improved, follow these guidelines:
   
The project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- Your HTML structure is quite good, but remember that this is a project where you have to show mastery using bootstrap if you need some information you can check the official [docs](https://getbootstrap.com/)
- In this project students must be more precise and to make sure that their page looks exactly the same as the source page, please modify typography, and add the missing sections, you can find the site [here](https://www.newsweek.com/)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @alvinlouis29
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindl and work on the suggested changes👇.
- [ ] Update readme file as previous TSE suggested
- [ ] Fix LInters errors
- [ ] Make sure your responsiveness on the medium and small screens.
**Explanation:** The cloned webpage is not responsive it should behave like the original. so please make sure its responsiveness like the original page

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
## Changes Requested # 
** HI @alvinlouis29  **
I'm really sorry to tell you that but your project needs a lot of work, you need to make look at least 95% like the original project, please make these following changes:
## Required #
- the live demo link is not working, make sure to fix it.
- the screenshot in the readme file is not working, make sure to fix it.
- the secondary and trending navbars should have the elements laid out in a row not as columns
- make sure not to use image duplicates
- the futured stories section is not responsive in small or mid sizes, please fix that
- make sure to read the inline comments

Please make sure not to file any further requests unless your clone looks like the original one, and behaves like it

if u need any further clarifications please contact me on slack @ mahmoud mohammad

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
Hello @alvinlouis29 
It seems you are having some difficulty understanding the requirements for this project.
This project has 2 main aims and they are to help solidify the knowledge of building a responsive webpage that you have gotten upon completion of the previous project and also to teach you how to use bootstrap in building responsive webpages. 
Therefore its is required that the page you are asked to clone be responsive just like the original page and that it also looks the same. That means it should be able to respond to different screen sizes just like it does in the original website and also looks like the original in the different screen sizes. 
I will try to highlight again all you need to do to get this project approved.
- [ ] First the screenshot in your readme is not loading, you have to fix this.
- [ ] The link to the live project does not point to the project. Please fix the link to load up the project when a user visits the link.
- [ ] Your read me has to be professional as stated in this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/), please update your readme accordingly.
- [ ] You also need to have a descriptive pull request message as described in this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
- [ ] Your nav should look like and behave like the original website nav across all screen sizes. I will implore that you study how the nav in the original website changes with different screen sizes and make your clone behave like that too.
- [ ] Use the same color and font as that used in the original website. Your clone needs to look exactly like the original website and that includes colors, fonts, etc. When I hover over some articles in your clone I am getting an underline with a blue color whereas orange color was used in the original.
- [ ] Your clone is missing a subscribe and newsletter section which can be found just before the footer in the original website. Please add these sections to your project.
- [ ] Generally, you need to make your clone responsive, as it is currently not responsive and breaks when the screen sizes start to resize.
- [ ] You should study how the original website responds to different screen sizes and make sure that your clone behaves just like that.

I implore you to not get discouraged, spend a little more time to learn about using bootstrap and you could also check on your previous project as a refresher on building response webpages, as your clone will have to look like the original and also has to be as responsive a the original before this project can be approved.
When you are done please request another review.
slack(@ebeagusamuel)

File: index.html
Comment (umairarshadbutt) [inline index.html]: Remove HTML commented code

File: index.html
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

File: index.html
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

File: index.html
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS from the landing page

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @alvinlouis29 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove **inline css** from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Remove HTML commentend code
- [x] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/100423121-4e88dd80-30ad-11eb-8aba-dca34268c43f.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/) ([screenshot](https://user-images.githubusercontent.com/29689273/94326203-fa358480-ffbb-11ea-81ec-b2704fb93212.png) from Pathwright)
- Navbar should be fixed

### Navbar (Medium screen)
- [x] Logo should be on most left side in navbar

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/alvinlouis29/Newsweek/features/index.html)|
|:---:|:---:|
|source-page|clone|


### Navbar on the small screen
- [x] Logo should be on most left side in navbar
- [x] Remove **LOGIN** 
- [x] Add user icon

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/alvinlouis29/Newsweek/features/index.html)|
|:---:|:---:|
|source-page|clone|



### SUBSCRIBE and NEWSLETTER sections
- [x] Design these two sections like the original 

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/alvinlouis29/Newsweek/features/index.html)|
|:---:|:---:|
|source-page|clone|


### Footer
- [x] `© 2019 NEWSWEEK DIGITAL LLC` is not managed on small screens. Please manage it

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/alvinlouis29/Newsweek/features/index.html)|
|:---:|:---:|
|source-page|clone|





##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: index.html
Comment (alvinlouis29) [reply index.html]: most of these changes i completed i don't know why they wont show in your browser the link to the live demo https://raw.githack.com/alvinlouis29/Newsweek/features/index.html is here please check it

File: index.html
Comment (alvinlouis29) [reply index.html]: thank you will remove the inline css

File: index.html
Comment (alvinlouis29) [reply index.html:180]: let me work on the changes you have requested and resubmit

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all project requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Comment by alvinlouis29** [comment]
project completed

**Pull request merged**
